packages/lambdas/src/__tests__/env-setup.ts

```diff
@@ -1,6 +1,8 @@
 import * as dotenv from "dotenv";
 import path from "path";
 const cwd = process.cwd();
-const paths = [cwd, ...(cwd.includes("packages") ? [] : ["packages", "lambdas"])];
+const paths = [cwd, ...(cwd.includes("packages") ? [] : ["packages", "api"])];
```

Comment:
was this intentionally changed to `api`?

Reply:
good catch, its a left-over/typo